docs/product-installation-guide/src/main/asciidoc/clustering-ks-runtime-standalone-proc.adoc

```diff
@@ -1,11 +1,39 @@
 [id='clustering-ks-runtime-standalone-proc']
-= Clustering {KIE_SERVER} for a standalone runtime environment
+= Clustering {KIE_SERVER} for a runtime environment
+In a runtime environment, {KIE_SERVER} runs rules and processes that support business decisions. The primary benefit of clustering a {KIE_SERVER} runtime environment is load balancing. If activity on one node of the cluster increases, that activity can be shared among the other nodes of the cluster to improve performance.
 
 .Prerequiste
-* An NFS server with a mounted partition accessible to an {EAP} user is available.
+* An NFS server with a mounted partition accessible to a {EAP} user is available.
 
 .Procedure
 . Set up a {EAP} 7.1 cluster as described in  https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/configuring_a_red_hat_jboss_eap_7_cluster/[_Configuring a {EAP} 7 Cluster_].
-. Install and configure Elasticsearch as described in <<clustering-elasticsearch-proc_{context}>>.
-. Install and configure AMQ Broker as described in <<clustering-artemis-activate-proc_{context}>>.
-. Install and configure {CENTRAL} as described in <<clustering-kie-server-install-proc_{context}>>.
+. Install {KIE_SERVER} on each node of the cluster as decribed in _Installing and configuring {PRODUCT} on {EAP} 7.1_.
+. If your environment is connected to Smart Router, on the java process running Smart Router do the following:
```

Comment:
Cap "Java". To avoid the "following:" issue in IBM (if you care), I'd reword "do the following on the Java process running Smart Router:" Both suggestions here apply to the next step also.